pkg/version/version.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2019 The Volcano Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+)
+
+var (
+	// Version shows the version of kube batch.
+	Version = "Not provided."
+	// GitSHA shoows the git commit id of kube batch.
+	GitSHA = "Not provided."
+	// Built shows the built time of the binary.
+	Built = "Not provided."
+)
+
+// PrintVersionAndExit prints versions from the array returned by Info() and exit
+func PrintVersionAndExit() {
+	apiVersion := "v1alpha1"
```

Comment:
Should be moved into a seperated section outside of the function.

Reply:
moved